models-info/src/main/java/com/flipkart/fdp/ml/transformer/VectorAssemblerTransformer.java

```diff
@@ -20,27 +20,44 @@ public VectorAssemblerTransformer(final VectorAssemblerModelInfo modelInfo) {
     }
 
     private double[] predict (Object[] inputs) {
-        List<Double> output = new ArrayList<>();
+
+        int vectorSize = Stream.of(inputs)
```

Comment:
can't we just use inputs.length here? For types other than double or double[], this method will lead to a RuntimeException anyway.